RadixWallet/Features/DappInteractionFeature/Coordinator/DappInteractionFlow.swift

```diff
@@ -146,6 +146,7 @@ struct DappInteractionFlow: Sendable, FeatureReducer {
 					.ifCaseLet(/MainState.chooseAccounts, action: /MainAction.chooseAccounts) {
 						AccountPermissionChooseAccounts()
 					}
+				EmptyReducer()
```

Comment:
intentional? is there some kind of contraint on only being able to do 3 `ifCaseLet`? Sounds arbitrary?

Hmm... should we not use Scope here btw??

Reply:
The problem here is that type checking takes too long, with the new overloads.

Reply:
But yes, Scope also works, so I switched to the new CaseKeyPath version of Scope